deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -34,6 +34,61 @@ kube-prometheus-stack:
       ##
       podMonitorSelectorNilUsesHelmValues: false
 
+      ## Number of replicas of each shard to deploy for a Prometheus deployment.
+      ## Number of replicas multiplied by shards is the total number of Pods created.
+      ##
+      replicas: 1
+
+      ## Prometheus StorageSpec for persistent data
+      ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/storage.md
+      ##
+      storageSpec:
+        {}
+        # volumeClaimTemplate:
+        #   metadata:
+        #     name: prom
+        #   spec:
+        #     storageClassName: cstorsc-ooqun
+        #     accessModes: ["ReadWriteOnce"]
+        #     resources:
+        #       requests:
+        #         storage: 10Gi
+        #   selector: {}
+
+  alertmanager:
+    ## Deploy alertmanager
+    ##
+    enabled: true
+
+    alertmanagerSpec:
+      ## Size is the expected size of the alertmanager cluster. The controller will eventually make the size of the
+      ## running cluster equal to the expected size.
+      replicas: 1
+
+      ## Storage is the definition of how storage will be used by the Alertmanager instances.
+      ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/storage.md
+      ##
+      storage:
+        {}
+        # volumeClaimTemplate:
+        #   metadata:
+        #     name: alert
+        #   spec:
+        #     storageClassName: cstorsc-ooqun
+        #     accessModes: ["ReadWriteOnce"]
+        #     resources:
+        #       requests:
+        #         storage: 10Gi
+        #   selector: {}
+
+  prometheusOperator:
```

Comment:
Why is this necessary?

Reply:
Yes, we can have this as this is a dependent chart.

Reply:
I meant what's the significance of placing it in our `values.yaml`. Is this updated, modified or used somewhere in our chart?

Reply:
Yes, I think we should put the dependent chart enabled field in values.yaml. Although while using persistence storage `tls` has to be enabled. That's what I have added it.

Reply:
Okay.